kubernetes/zenko/values.yaml

```diff
@@ -72,7 +72,7 @@ reporting:
   enabled: true
   successfulJobsHistory: 1
   countItems:
-    schedule: "* */1 * * *"
+    schedule: "@hourly"
```

Comment:
I like this and didn't know about this cron format. It is much more interpretable!